src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py

```diff
@@ -343,16 +343,18 @@ def get_chart_mapping_schema(
         
         logger.debug("Get chart mapping schema for %s", helm_package.name)
         
-        non_def_values = os.path.join(
-            self._tmp_folder_name, helm_package.name, "values.mappings.yaml"
-        )
+        non_def_values = helm_package.path_to_mappings
```

Comment:
Why is this called `non_def_values`? It looks to me like this should just be called path_to_mappings or something like that. Without knowing anything about CNFs I have no idea what this variable is supposed to represent, but maybe it is obvious for someone who knows CNFs

Reply:
You're right. It's leftover from when the mappings file was called values.nondefs. Will change